CHANGES.txt

```diff
@@ -1,3 +1,7 @@
+Next
+----
+- fetch Landsat panchromatic band min/max (#58)
+
```

Comment:
The bug fix here is that band 8 was missing, yes? Let's say so in the change log.

Sorry to be a pain!

Reply:
I'll re-phrase